mlflow/models/evaluation/base.py

```diff
@@ -775,81 +775,157 @@ def _start_run_or_reuse_active_run():
         yield active_run.info.run_id
 
 
-def _normalize_evaluators_and_evaluator_config_args(
-    evaluators,
-    evaluator_config,
-):
+def _resolve_default_evaluator(model_type, evaluator_config):
+    """
+    Determine which built-in evaluators should be used for the given model type by default.
+
+    Previously, MLflow evaluate API only had a single "default" evaluator used for all models like
+    classifier, regressor, etc. We split it into multiple built-in evaluators for different model
+    types for maintainability, but in order to maintain backward compatibility, we need to map
+    the "default" provided by users to the correct built-in evaluators.
+
+    Args:
+        model_type: A string describing the model type (e.g., "regressor", "classifier", …).
+        evaluator_config: A dictionary of additional configurations for the evaluator.
+    """
     from mlflow.models.evaluation.evaluator_registry import _model_evaluation_registry
 
+    builtin_evaluators = []
+    for evaluator_name in list(_model_evaluation_registry._registry.keys()):
```

Comment:
```suggestion
    for name, evaluator in list(_model_evaluation_registry._registry.items()):
```

can we use `.items` here?

Reply:
Yup! (we also don't need `list`)